1-Authentication/2-sign-in-angular/SPA/package-lock.json

```diff
@@ -0,0 +1,13890 @@
+{
+  "name": "1-sign-in-angular",
```

Comment:
Holy moly there's a lot going on in this deps file. Is this all just Angular stuff we can't remove?

Reply:
As far as I know, yes.